app/containers/main-container.jsx

```diff
@@ -3,16 +3,46 @@ import { connect } from 'react-redux'
 
 /* components */
 import GoalsList from './../components/goals-list/goals-list.jsx';
+import AddGoal from './../components/add-new-goal/add-new-goal.jsx';
+
+/* ations */
+import action_stepAddGoal from './../actions/goals-list.js';
+
 
 const MainContent = props => {
+
+  let component;
+
+  switch (props.step.currentStep) {
```

Comment:
This is wicked. People often use react router to do this kind of thing, but it adds lots of complexity and I hate it. This is nice and simple + obvious